python/cudf_polars/cudf_polars/dsl/expr.py

```diff
@@ -811,6 +823,19 @@ def do_evaluate(
                     else prefix.obj,
                 )
             )
+        elif self.name == pl_expr.StringFunction.Replace:
+            column, target, repl = columns
+            n, literal = self.options
+            return Column(
+                plc.strings.replace.replace(
+                    column.obj, target.obj_scalar, repl.obj_scalar, maxrepl=n
+                )
```

Comment:
Polars suports the case where `target` is a scalar, but `repl` is a list of strings (one per row in `column`).

But libcudf does not support this.

Can we complain early? I think it's OK if we raise if `target` and `repl` are not from `expr.Literal` objects.

Reply:
Done.